Formula/moira.rb

```diff
@@ -0,0 +1,45 @@
+class Moira < Formula
+  desc "Clients for the Athena Service Management System"
+  homepage "https://debathena.mit.edu"
+  url "https://github.com/mit-athena/moira/archive/4.0.0.3+51+g65d55c5.tar.gz"
+  sha256 "d418681ae4ec61a124c55fb67a6bba0d89b59dc760c4d1c9a1e8e1c9c7b69b19"
+
+  depends_on "e2fsprogs" # for com_err, as recommended by Debian packaging
+  depends_on "hesiod"
+  depends_on "homebrew/dupes/krb5"
```

Comment:
Fixed, by using options. Would this be acceptable?

> can't depend on dupes or other **taps**

Wait, so you're saying I can't use homebrew-core either!? :^)